code/modules/admin/topic.dm

```diff
@@ -1456,6 +1456,9 @@
 		usr << browse(dat.Join("<br>"), "window=related_[C];size=420x300")
 
 	else if(href_list["centcomlookup"])
+		if(href_list) // BUBBER EDIT BEGIN - Crash Bandaid
```

Comment:
Also, href_list is guaranteed to have the key "centcomlookup", does that not ensure this condition is always true? I don't know how truthiness works with regards to lists.

Reply:
> Also, href_list is guaranteed to have the key "centcomlookup", does that not ensure this condition is always true? I don't know how truthiness works with regards to lists.

If anything is in the contents of a variable its true. Theres some weird null differences.

Reply:
Also it's intentionally like this as it doesn't make the linter cry like just adding a return does.

Reply:
nonissue bc of linter context